mappings/net/minecraft/world/gen/carver/Carver.mapping

```diff
@@ -82,3 +84,4 @@ CLASS net/minecraft/class_2939 net/minecraft/world/gen/carver/Carver
 		ARG 3 scaledRelativeY
 		ARG 5 scaledRelativeZ
 		ARG 7 y
+	METHOD method_28616 getCodec ()Lcom/mojang/serialization/Codec;
```

Comment:
```suggestion
	METHOD method_28616 getConfigCodec ()Lcom/mojang/serialization/Codec;
```

Reply:
This is a codec that serialises a configured carver.

Reply:
oops, yeah, went a bit too fast on the suggestions.